deploy/cr.yaml

```
@@ -447,7 +447,7 @@ spec:
 
   backup:
     enabled: true
-    image: perconalab/percona-server-mongodb-operator:main-backup
+    image: perconalab/percona-server-mongodb-operator:2.0.4
```

we don't have this image. should we use the official image?

```suggestion
    image: percona/percona-backup-mongodb:2.0.4
```